launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml

```diff
@@ -9,7 +9,8 @@
   <arg name="classifier_model_path" default="$(find-pkg-share traffic_light_classifier)/data/traffic_light_classifier_mobilenetv2_batch_6.onnx" description="classifier onnx model path"/>
   <arg name="input/cloud" default="/sensing/lidar/top/pointcloud_raw" description="point cloud for occlusion prediction"/>
   <arg name="internal/traffic_signals" default="/perception/traffic_light_recognition/internal/traffic_signals"/>
-  <arg name="output/traffic_signals" default="/perception/traffic_light_recognition/traffic_signals"/>
+  <arg name="external/traffic_signals" default="/perception/traffic_light_recognition/external/traffic_signals"/>
+  <arg name="output/traffic_signals" default="/perception/traffic_light_arbiter/traffic_signals"/>
```

Comment:
[Q]
Which node request "/perception/traffic_light_arbiter/traffic_signals" as output of perception? This change (output I/F of perception) doesn't sound correct to me
Which node publish "/perception/traffic_light_recognition/external/traffic_signals"?

Reply:
@miursh 
I have corrected the output topic name as per your suggestion. :bow: 
https://github.com/autowarefoundation/autoware.universe/pull/4393/commits/308a641b6e1c9a061b1cc96dc8cfac43d80d1a7e

As for external/traffic_signals, I just created an argument for it. There was no major change intended.

Reply:
I will create a separate PR to change the input topic of the behavior_velocity module.